reference/checkouts.v3.yml

```diff
@@ -9428,6 +9436,12 @@ components:
         custom_checkout_supports_uco_settings:
           type: boolean
           description: Boolean value that specifies whether this checkout supports Optimized One-Page Checkout settings.
+        custom_checkout_sri_hash:
+          type: string
+          description: 'The Subresource Integrity (SRI) hash for the custom checkout script URL.'
+        custom_order_confirmation_sri_hash:
+          type: string
+          description: 'The Subresource Integrity (SRI) hash for the custom order confirmation script URL.'
```

Comment:
are those two fields in the request required? if so or not, maybe added `required`: true/false

Reply:
These fields are optional. In the Swagger doc, only mandatory fields should be listed under the required section. There is no need to explicitly add `required: true/false` for each field.